certified-connectors/Tesseron ASM Basic Data/apiDefinition.swagger.json

```diff
@@ -20,321 +20,6 @@
-                  "format": "int32",
-                  "description": "The Tesseron ID of the requested enterprise.",
-                  "title": "Enterprise ID"
-                },
-                "EnterpriseName": {
-                  "type": "string",
-                  "description": "The name of the requested enterprise.",
-                  "title": "Enterprise Name"
-                },
-                "BusinessPartnerId": {
-                  "type": "string",
-                  "description": "The business partner ID of the requested enterprise.",
-                  "title": "Business Partner ID"
-                },
-                "MailDomain": {
-                  "type": "string",
-                  "description": "The mail domain of the requested enterprise.",
-                  "title": "Mail Domain"
-                },
-                "Message": {
-                  "type": "string",
-                  "description": "Message"
-                },
-                "Success": {
-                  "type": "boolean",
-                  "description": "Success"
-                },
-                "Status": {
-                  "type": "string",
-                  "description": "Status"
-                }
-              }
-            }
-          },
-          "303": {
-            "description": "Endpoint is obsolete. Please use the endpoint GetEnterprises",
-            "schema": {
-              "type": "object",
-              "properties": {
-                "Message": {
-                  "type": "string",
-                  "description": "Message"
-                },
-                "Success": {
-                  "type": "boolean",
-                  "description": "Success"
-                },
-                "EnterpriseId": {
-                  "type": "array",
-                  "items": {
-                    "type": "integer",
-                    "format": "int32"
-                  },
-                  "title": "Enterprise ID of the requested business partner."
-                }
-              }
-            }
-          }
-        },
-        "summary": "[Obsolete] Get enterprise information",
-        "operationId": "GetEnterprise",
-        "description": "Get the main enterprise information (enterprise ID, business partner ID & domain)",
-        "parameters": [
-          {
-            "name": "body",
-            "in": "body",
-            "required": true,
-            "schema": {
-              "type": "object",
-              "properties": {
-                "enterpriseId": {
-                  "type": "integer",
-                  "format": "int32",
-                  "description": "Please provide the ID of the business partner.",
-                  "title": "Enterprise ID"
-                }
-              },
-              "required": [
-                "enterpriseId"
-              ]
-            }
-          }
-        ]
-      }
-    },
-    "/ApiEnterprise/SearchEnterpriseByParameter": {
```

Comment:
Can you please confirm that the existing flows will not break the flows?

The operation has been deleted. This change breaks existing flows. ValuePath: operation/SearchEnterpriseByParameter

Reply:
We will inform our customers, that this endpoint is no longer supported